r-package/inst/include/models/adult_hiv_model_simulation.hpp

```diff
@@ -136,10 +137,15 @@ struct AdultHivModelSimulation<Config> {
                               (n_ha.h_hiv_adult(hm, ha, g) + i_ha.artpop_hahm);
           }
 
-          i_ha.deaths = i_ha.cd4mx_scale * p_ha.cd4_mortality(hm, ha, g) * n_ha.h_hiv_adult(hm, ha, g);
-          i_ha.p_hiv_deaths_age_sex(ha, g) += opts.dt * i_ha.deaths;
-          n_ha.h_hiv_deaths_no_art(hm, ha, g) += opts.dt * i_ha.deaths;
-          i_ha.grad(hm, ha, g) = -i_ha.deaths;
+          i_ha.deaths_hiv = i_ha.cd4mx_scale * p_ha.cd4_mortality(hm, ha, g) * n_ha.h_hiv_adult(hm, ha, g);
+          i_ha.h_hiv_deaths_age_sex(ha, g) += opts.dt * i_ha.deaths_hiv;
+          n_ha.h_hiv_deaths_no_art(hm, ha, g) += opts.dt * i_ha.deaths_hiv;
+
+	  i_ha.deaths_excess_nonaids = p_ha.cd4_nonaids_excess_mort(hm, ha, g) * n_ha.h_hiv_adult(hm, ha, g);
+	  i_ha.h_deaths_excess_nonaids_agesex(ha, g) += opts.dt * i_ha.deaths_excess_nonaids;
```

Comment:
I think would be best to align formatting between i_ha.h_deaths_excess_nonaids_agesex and  i_ha.h_hiv_deaths_age_sex. I think I prefer i_ha.h_excess_nonaids_deaths_age_sex and  i_ha.h_hiv_deaths_age_sex (mostly just on the ordering, don't have an opinion on age_sex vs agesex)